### DISCUSSION TITLE
Pulling a specified row of data based on a matching value

### DISCUSSION THREAD

**Discussion by dwchan69** [dfinke/ImportExcel#1571] [General]
Import-Excel load and import all rows within an XLS worksheets.  Curiously, can you tell it to only pull in a specified row based on a match?  Let say you have a row call small, a row call medium, another row call large.  is there a way to control import-excel to only pull in row where say the 1st column is call "Small"?

**Comment by dfinke** [comment]
Check out. You need additional drivers installed.

https://github.com/dfinke/ImportExcel/blob/ab526558a3cd643e10cad3c04f35e3645389d857/Public/Invoke-ExcelQuery.ps1
